Sources/Testing/Testing.docc/AddingTags.md

```diff
@@ -117,36 +117,36 @@ any test target and applied to any test:
 - ``Tag/blue``
 - ``Tag/purple``
 
-The testing library does not assign any semantic meaning to these tags, nor does
+The testing library doesn't assign any semantic meaning to these tags, nor does
 the presence or absence of these tags affect how the testing library runs tests.
 
 ## Customize a tag's appearance
 
-By default, a tag does not appear in a test's output when the test is run. It is
+By default, a tag doesn't appear in a test's output when the test is run. It's
 possible to assign colors to tags defined in a package so that when the test is
 run, the tag is visible in its output.
 
 To add colors to tags, create a directory in your home directory named
-`".swift-testing"` and add a file named `"tag-colors.json"` to it. This file
-should contain a JSON object (a dictionary) whose keys are strings representing
+`.swift-testing` and add a file named `tag-colors.json` to it. This file
+should contain a JSON object (a dictionary) whose keys are strings that represent
 tags and whose values represent tag colors.
 
-- Note: On Windows, create the `".swift-testing"` directory in the
-  `"AppData\Local"` directory inside your home directory instead of directly
+- Note: On Windows, create the `.swift-testing` directory in the
+  `AppData\Local` directory inside your home directory instead of directly
   inside it.
 
 Tag colors can be represented using several formats:
 
-- The strings `"red"`, `"orange"`, `"yellow"`, `"green"`, `"blue"`, or
-  `"purple"`, representing corresponding predefined instances of ``Tag``, i.e.
+- The strings `red`, `orange`, `yellow`, `green`, `blue`, or
+  `purple`, represent corresponding predefined instances of ``Tag``, like
   ``Tag/red``, ``Tag/orange``, ``Tag/yellow``, ``Tag/green``, ``Tag/blue``, and
-  ``Tag/purple``;
-- A string of the form `"#RRGGBB"`, containing a hexadecimal representation of
-  the color in a device-independent RGB color space; or
-- The `null` literal value, representing "no color."
+  ``Tag/purple``.
+- A string of the form `#RRGGBB`, containing a hexadecimal representation of
+  the color in a device-independent RGB color space.
+- The `null` literal value, which represents no color.
 
-For example, to set the color of the tag `"critical"` to orange and the color of
-the tag `.legallyRequired` to teal, the contents of `"tag-colors.json"` can
+For example, to set the color of the `critical` tag to orange and the color of
```

Comment:
`critical` here is a string literal and needs the quotes.

Reply:
Will revert.